src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l 0;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: $font-size-l;
+  line-height: 1.333;
+}
+
+.SecondaryHero-message-description {
+  padding: $padding-page 0;
+}
+
+.SecondaryHero-modules {
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(3, 1fr);
+  width: 70%;
+}
+
+.SecondaryHero-module {
+  align-items: center;
+  background-color: $white;
+  border-radius: $border-radius-s;
+  display: flex;
+  flex-direction: column;
+  // flex-grow: 0;
+  font-size: 18px;
+  // justify-content: space-around;
+  line-height: 1.222;
+  // margin: 16px 0;
+  padding: 0 16px 16px 16px;
+  // position: relative;
+  text-align: center;
```

Comment:
I think you can remove this now that you have `align-items`, right?

Reply:
I can, but only if I move the rule into a selector for `SecondaryHero-module-description`, because it was actually needed for this. So I've done that as that's more explicit.